app/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdk 30
+    buildToolsVersion "31.0.0"
+
+    defaultConfig {
+        applicationId "ir.divar"
+        minSdk 21
```

Comment:
Reduce this version to bazaarVersion minSdk

Reply:
Done.